src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/CaseClientServiceException.java

```diff
@@ -3,15 +3,14 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.client.HttpStatusCodeException;
 
-public class CaseTransformationException extends Exception {
-
+public class CaseClientServiceException extends Exception {
```

Comment:
'client service'?

Reply:
Yes, thrown due to an error calling a client service. There used to be just one call - transform, now there is one more - update, so it is renamed to a more generic.